internal/poadraftapplication/poa_draft_application.go

```diff
@@ -0,0 +1,100 @@
+package poadraftapplication
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+
+	"github.com/ministryofjustice/opg-search-service/internal/response"
+)
+
+const AliasName = "poadraftapplication"
+
+type DraftApplicationDonor struct {
+	Name string `json:"name"`
+	Dob string `json:"dob"`
+	Postcode string `json:"postcode"`
+}
+
+type DraftApplication struct {
+	UID *string `json:"uId"`
```

Comment:
Could this be a direct rather than a pointer?
```suggestion
	UID string `json:"uId"`
```